app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -302,6 +303,7 @@ public void onExternalResponse(GeckoSession session, GeckoSession.WebResponseInf
                 @Override
                 public void onCrash(GeckoSession session) {
                     Log.i(TAG, "Crashed, opening new session");
+                    Sentry.capture("GeckoSession crashed, opening new session");
```

Comment:
Let's move all calls to Sentry into SentryWrapper so that we have a single point where we can control what gets send when.